Bootloaders/HID/HostLoaderApp/Makefile

```diff
@@ -7,7 +7,8 @@ ifeq ($(OS), LINUX)  # also works on FreeBSD
 CC ?= gcc
 CFLAGS ?= -O2 -Wall
 hid_bootloader_cli: hid_bootloader_cli.c
-	$(CC) $(CFLAGS) -s -DUSE_LIBUSB -o hid_bootloader_cli hid_bootloader_cli.c -lusb
+	#$(CC) $(CFLAGS) -s -DUSE_LIBUSB -o hid_bootloader_cli hid_bootloader_cli.c ../AES/aes.c -lusb
+	$(CC) $(CFLAGS) -s -DUSE_HIDAPI -o hid_bootloader_cli hid_bootloader_cli.c ../AES/aes.c -I/usr/include/hidapi/ -lhidapi-libusb
```

Comment:
Oh good point. That was because I modified the Host app. You have to remove it of course.